migrations/1_create_asset_allocations_up.sql

```diff
@@ -0,0 +1,9 @@
+    CREATE TABLE asset_allocations(
+        asset_id varchar (100) ,
+        user_id varchar (100),
+        allocated_by varchar(100) NOT NULL,
+        isactive boolean NOT NULL,
+        from_date Date NOT NULL,
+        to_date Date,
+        PRIMARY KEY(asset_id,user_id) 
```

Comment:
table should have created_at & updated_at fields with default set to current time stamp , this fields are very crucial, pls add in all the tables